package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,347 @@
+eigenvectors of this matrix.
+
+For each eigenvector, its eigenvalue is the variance that the eigenvector
+explains. Stored in :attribute:`cumulated_variance`, a ratio for each number of
+eigenvectors up to index i is provided to quickly find out how many principal
+components are needed to explain the amount of variance reflected by those i
+eigenvectors. For most data, :attribute:`cumulated_variance` will be
+approximately equal to one for some n that is significantly smaller than the
+total number of components, these are the components of interest given
+by Principal Component Analysis.
+
+From here, we can project a trajectory onto these principal components and
+attempt to retrieve some structure from our high dimensional data.
+
+For a basic introduction to the module, the :ref:`PCA-tutorial` shows how
+to perform Principal Component Analysis.
+
+.. _PCA-tutorial:
+The example uses files provided as part of the MDAnalysis test suite
+(in the variables :data:`~MDAnalysis.tests.datafiles.PSF` and
+:data:`~MDAnalysis.tests.datafiles.DCD`). This tutorial shows how to use the
+PCA class.
+
+First load all modules and test data ::
+    >>> import MDAnalysis as mda
+    >>> import MDAnalysis.analysis.pca as pca
+    >>> from MDAnalysis.tests.datafiles import PSF, DCD
+
+Given a universe containing trajectory data we can perform PCA using
+:class:`PCA`:: and retrieve the principal components.
+    >>> u = mda.Universe(PSF,DCD)
+    >>> PSF_pca = pca.PCA(u, select='backbone')
+    >>> PSF_pca.run()
+
+Inspect the components to determine the principal components you would like
+to retain. The choice is arbitrary, but I will stop when 95 percent of the
+variance is explained by the components. This cumulated variance by the
+components is conveniently stored in the one-dimensional array attribute
+`cumulated_variance`. The value at the ith index of `cumulated_variance` is the
+sum of the variances from 0 to i.
+
+    >>> n_pcs = np.where(PSF_pca.cumilated_var > 0.95)[0][0]
+    >>> atomgroup = u.select_atoms('backbone')
+    >>> pca_space = PSF_pca.transform(atomgroup, n_components=n_pcs)
+
+From here, inspection of the pca_space and conclusions to be drawn from the
+data are left to the user.
+
+Functions
+---------
+.. autoclass:: PCA
+.. autofunction:: cosine_content
+"""
+from six.moves import range
+import logging
+import warnings
+
+import numpy as np
+
+from scipy.integrate import simps
+from MDAnalysis.core import AtomGroup
+from MDAnalysis import Universe
+from MDAnalysis.analysis.align import _fit_to
+from MDAnalysis.lib.log import ProgressMeter
+
+from .base import AnalysisBase
+
+
+class PCA(AnalysisBase):
+    """Principal component analysis on an MD trajectory.
+
+    After initializing and calling method with a universe or an atom group,
+    principal components ordering the atom coordinate data by decreasing
+    variance will be available for analysis. Please refer to the
+    :ref:`PCA-tutorial` for more detailed instructions.
```

Comment:
Sorry this doesn't work in an interactive session. You have to do a simple code example.
Something like this is enough to get a very short overview. It is still good to link to the tutorial but that is only seen in the online docs as a link.

```python
pca = PCA(atomgroup, select='backbone').run()
pca_space =  pca.transform(atomgroup.select_atoms('backbone'), 3)
```